src/dosbox_splash.svg

```diff
@@ -0,0 +1,189 @@
+     style="display:inline"
+     sodipodi:insensitive="true">
+    <rect
+       style="opacity:1;fill:#ba3d00;fill-opacity:1;fill-rule:evenodd;stroke:none;paint-order:fill markers stroke"
+       id="rect5227"
+       width="264.58334"
+       height="264.58334"
+       x="0"
+       y="32.416653" />
+  </g>
+  <g
+     inkscape:groupmode="layer"
+     id="layer2"
+     inkscape:label="Box"
+     style="display:inline"
+     sodipodi:insensitive="true">
+    <g
+       sodipodi:type="inkscape:box3d"
+       id="g4632"
+       style="display:inline;fill:#1f1f1f;fill-opacity:1;stroke:none"
+       inkscape:perspectiveID="#perspective4588-6"
+       inkscape:corner0="2.6212412 : 0.22433035 : 0 : 1"
+       inkscape:corner7="2.0232549 : 0.11324788 : 0.87579633 : 1"
+       inkscape:export-xdpi="37.150002"
+       inkscape:export-ydpi="37.150002">
+      <path
+         sodipodi:type="inkscape:box3dside"
+         id="path4634"
+         style="fill:#1f1f1f;fill-opacity:0.1;fill-rule:evenodd;stroke:none;stroke-linejoin:round"
+         inkscape:box3dsidetype="6"
+         d="M 74.083397,68.065216 V 178.51117 L 155.40573,167.62798 V 78.691335 Z"
+         points="74.083397,178.51117 155.40573,167.62798 155.40573,78.691335 74.083397,68.065216 " />
+      <path
+         sodipodi:type="inkscape:box3dside"
+         id="path4642"
+         style="fill:#1f1f1f;fill-opacity:0.1;fill-rule:evenodd;stroke:none;stroke-linejoin:round"
+         inkscape:box3dsidetype="13"
+         d="m 74.083397,178.51117 29.924673,12.4683 87.0729,-15.35291 -35.67524,-7.99858 z"
+         points="104.00807,190.97947 191.08097,175.62656 155.40573,167.62798 74.083397,178.51117 " />
+      <path
+         sodipodi:type="inkscape:box3dside"
+         id="path4644"
+         style="fill:#1f1f1f;fill-opacity:0.1;fill-rule:evenodd;stroke:none;stroke-linejoin:round"
+         inkscape:box3dsidetype="11"
+         d="m 155.40573,78.691335 35.67524,-5.641384 V 175.62656 l -35.67524,-7.99858 z"
+         points="191.08097,73.049951 191.08097,175.62656 155.40573,167.62798 155.40573,78.691335 " />
+      <path
+         sodipodi:type="inkscape:box3dside"
+         id="path4636"
+         style="fill:#1f1f1f;fill-opacity:0.1;fill-rule:evenodd;stroke:none;stroke-linejoin:round"
+         inkscape:box3dsidetype="5"
+         d="m 74.083397,68.065216 29.924673,-9.377417 87.0729,14.362152 -35.67524,5.641384 z"
+         points="104.00807,58.687799 191.08097,73.049951 155.40573,78.691335 74.083397,68.065216 " />
+      <path
+         sodipodi:type="inkscape:box3dside"
+         id="path4640"
+         style="fill:#1f1f1f;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-linejoin:round"
+         inkscape:box3dsidetype="14"
+         d="M 104.00807,58.687799 V 190.97947 l 87.0729,-15.35291 V 73.049951 Z"
+         points="104.00807,190.97947 191.08097,175.62656 191.08097,73.049951 104.00807,58.687799 "
+         inkscape:export-xdpi="37.150002"
+         inkscape:export-ydpi="37.150002" />
```

Comment:
Comment only - this is very interesting; I haven't done much with putting art into code. I'm assuming this is some kind of SVG reprensentation? Is there a link to the source art somewhere, and the tool used to go from SVG to this codified-notation? (sorry if I'm just not seeing it.. pages of XML tend to put me in a trance ;-) )